config/config.go

```diff
@@ -993,6 +993,7 @@ type KubernetesSDConfig struct {
 	BearerToken     string         `yaml:"bearer_token,omitempty"`
 	BearerTokenFile string         `yaml:"bearer_token_file,omitempty"`
 	TLSConfig       TLSConfig      `yaml:"tls_config,omitempty"`
+	Namespaces      []string       `yaml:"namespaces,omitempty"`
```

Comment:
We should make sure that we could still add discovery here, so I'm thinking of something like a `NamespaceDiscovery` here instead of `[]string`, which in turn has the field `static_namespaces` or something in that direction, where if we want to add more dynamic discovery can add an additional field to do that in the `NamespaceDiscovery` struct.